tutorials/generating-synthetic-data/synthetic-data-generation-5.md

```diff
@@ -112,39 +85,33 @@ $ inductiva storage list -m 4
 
 ```
 
-## Impact of Particle Radius on Simulation Volume and Data Output
+## Checking the volume of data produced
 
-The table below shows how the different particle radius values we've chosen affect
-the total number of particles and the amount of data generated. As we decrease
-the particle radius, we need more particles for the simulation, increasing the
-data volume. Specifically, halving the particle radius results in an eightfold
-increase in the number of particles and the data produced, due to the volume
-scaling with the cube of the particle radius ($$r_{particle}{^3}$$).
+The table below shows how different particle radius affect the total number of particles and the amount of data generated by the simulation (4 seconds od simulated time). As we decrease the particle radius, we need more particles to fill in the corresponding volumes in the simulation, increasing the amount of data we generate. Specifically, halving the particle radius results in an eightfold
+increase in the number of particles, as expected. 
 
 <span class="mt-0 block sm:text-left text-base"><strong>Table 1.</strong> Number
 of particles and total size of simulation output for each particle radius.</span>
 
-| Particle Radius | Total N. of Particles | Size of data produced  |
+| Particle Radius | Total N. of Particles | MB of data produced  |
 |-----------------|-----------------------|------------------------|
-| 0.01            | 15625                 | 166 MB                 |
-| 0.008           | 29791                 | 213 MB                 |
-| 0.006           | 73720                 | 525 MB                 |
-| 0.004           | 244584                | 1.76 GB                |
+| 0.01            | 15625                 | 166  |
+| 0.008           | 29791                 | 213  |
+| 0.006           | 73720                 | 525  |
+| 0.004           | 244584                | 1.76 |
```

Comment:
Here, you moved the MB to the header of the table, but the last row was in GB, so we need to update the number to be 1000x bigger, so 1760. (Note: I think that when people use bases of 2 for storage units, i.e. 1024 factor instead of 1000, the notation is MiB instead of MB -- so here I am assuming the storage unit is using powers of 10 -- source: https://en.wikipedia.org/wiki/Byte#Multiple-byte_units)

Reply:
You are correct! Changing.